### PR TITLE
plink2: depends on openblas if it is Linux

### PR DESCRIPTION
**Pull request by tmozgach**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

**Comment by tmozgach** [comment]
@sjackman could you please merge?

File: plink2.rb [hunk 1]
```diff
@@ -1,23 +1,30 @@
 class Plink2 < Formula
+  desc "It's purely on analysis of genotype/phenotype data"
```
Comment (sjackman) [inline plink2.rb]: Please change the description to "Analyze genotype and phenotype data"

File: plink2.rb [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+  homepage "https://www.cog-genomics.org/plink2"
   url "https://github.com/chrchang/plink-ng/archive/v1.90b3.tar.gz"
   version "1.90b3"
   # doi "10.1186/s13742-015-0047-8"
   # tag "bioinformatics"
-  homepage "https://www.cog-genomics.org/plink2"
   sha256 "2f4afc193c288b13af4410e4587358ee0a6f76ed7c98dd77ca1317aac28adf0d"
   revision 1
```
Comment (sjackman) [inline plink2.rb]: Please bump this to `revision 2`.

**Comment by sjackman** [comment]
Thanks, Tanya!

**Comment by tmozgach** [comment]
@sjackman Done

**Comment by sjackman** [comment]
Merged. Thanks, Tanya!

**Pull request closed without merging**